easybuild/easyconfigs/l/libxc/libxc-4.0.1-intel-2017b.eb

```diff
@@ -14,7 +14,11 @@ toolchainopts = {'lowopt': True, 'optarch': False}
 
 source_urls = ['http://www.tddft.org/programs/octopus/down.php?file=libxc/%(version)s']
 sources = [SOURCE_TAR_GZ]
-checksums = ['836cb2f529bb9d1979a347d4d5460bf9a18c64f39b8127e88f3004471a72ab30']
+patches = ['libxc_4.0_longdouble.patch']
```

Comment:
@micaeljtoliveira Please rename to match with the 3.0 patch

Reply:
One patch should be applied to all the 3.x.x releases, while the other should be applied only to the 4.0.x releases. What should be the correct naming convention here?

Reply:
@micaeljtoliveira We don't have any easyconfigs for `libxc` 3.x where `x > 0`, so using `3.0` and `4.0` in the patch filename makes sense?

Reply:
@boegel You're absolutely right. I forgot there is no 3.1.x series (and probably there is never going to be one).